pkg/apis/maistra/conversion/policy.go

```diff
@@ -9,15 +9,15 @@ import (
 )
 
 // XXX: Not all of the settings are mapped correctly, as there are differences
-// between v1.0/v1.1 and v2.0
+// between v1.1 and v2.0
 
 func populatePolicyValues(in *v2.ControlPlaneSpec, values map[string]interface{}) error {
 	// Cluster settings
 	if in.Policy == nil {
 		return nil
 	}
 
-	istiod := !(in.Version == "" || in.Version == versions.V1_0.String() || in.Version == versions.V1_1.String())
+	istiod := !(in.Version == "" || in.Version == versions.V1_1.String())
```

Comment:
nit: `istiod := in.Version != versions.V1_1.String()`

Reply:
not sure about this one. I don't want to break existing 1.0 SMCPs-- just not reconcile them anymore

Reply:
actually, I guess the fact that in other places we don't have the check for empty string means that it's fine